cli/src/commands/next.rs

```diff
@@ -128,12 +128,14 @@ pub(crate) fn cmd_next(
 
     let target_revset = if args.conflict {
         start_revset
+            .children()
```

Comment:
nit: this could be `.descendants_at(args.offset)` to make these queries look similar, but I don't know if it's better.

Reply:
I don't think its clearer as `children()` are always the first descendants. But if you replace it, add a comment that it is equivalent to `children()`

Reply:
I think `children()` is clearer, so I'll leave it as is.